### PR TITLE
add information on all fragments and error pages in README

### PR DESCRIPTION
**Pull request by Joseph-Nash**
Added information about each fragment and error page in the README.

__CPS_212__

### Type of change

* [] Bug fix
* [x] New feature
* [] Breaking change

### Pull request checklist

* [] I have added unit tests for new code that I have added
* [] I have added/updated functional tests where appropriate
* [x] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,38 @@ Projects using this must implement changes to the ```baseLayout.html``` file.
 ## Description 
 
 A common-web dependency to contain reusable resources and fragments for SpringBoot web development, e.g: back-buttons, headers, footers, continue buttons, etc.
+
+
+## More Info
+
+### backButtonLink.html
+
+Fragment that provides a button to go backwards in the journey.
```
Comment (BSaunders95) [inline README.md]: Is it worth detailing the parameters required for each fragment? For example, this one requires a `backButton` model attribute

File: README.md [hunk 2]
```diff
@@ -31,3 +31,38 @@ Projects using this must implement changes to the ```baseLayout.html``` file.
...
+
+Fragment that listens to user interactions. Contains a customisable field that needs to be set in the ```baseLayout.html```, as mentioned above.
+
+### userBar.html
+
+Fragment that contains several links and information for the user. Links to Your details, Your filings, Companies you follow and Sign out are available, as well as displaying the user's email address.
+
+### error.html
+
+Generic error page that gives the user an option to email Companies House.
```
Comment (BSaunders95) [inline README.md]: Is it worth noting the requirement of the template structure here? This one requires the baseLayout to exist under the `templates/layouts` directory

File: README.md (hunk 1 above)
Comment (Joseph-Nash) [reply README.md]: Mind if we talk about this tomorrow? It's a bit late for me to think straight 👍

Comment (BSaunders95) [reply README.md]: Haha of course, grab me when you get in 👍

**Pull request merged**